util/missingcrypto.txt

```diff
@@ -445,6 +445,7 @@ ERR_load_ENGINE_strings(3)
 ERR_load_ERR_strings(3)
 ERR_load_ESS_strings(3)
 ERR_load_EVP_strings(3)
+ERR_load_HTTP_strings(3)
```

Comment:
Hmmmm.....I'm not sure what we should do about these functions. Do new ones actually need to be public?

Reply:
Errrr..... this issue is not the point of this PR.
I know that _thy shall not add new functions to missingcrypto.txt_
but I did this here fully in line with existing practice for the other string loading functions.
I don't know if these need to be public (but I do know that `ERR_load_HTTP_strings()`) needs to be called on lib init, which I made sure),
and if they don't need to be public IMHO this should be done in some independent cleanup PR.

Reply:
Fair enough...my comment was more of a side note.

Reply:
Well, the cleanest solution would be to simply add in `doc/man3/ERR_load_crypto_strings.pod` one or two sentences generically for all these string loading functions.

On the other hand, apparently none of them need to be exported. 
For curiosity I just did a little experiment: remove all those specific `ERR_load_*_strings()`  functions from `util/libcrypto.um` and call `make test`. 
Nothing bad happened. Even `make doc-nits` is happy (also with the then redundant entries in `util/missingcrypto.txt`).